client/pages/home.jsx

```diff
@@ -1,8 +1,120 @@
-import React from 'react';
+/* eslint-disable multiline-ternary */
+
+import React, { useEffect, useState } from 'react';
+
+const App = () => {
+  const [venues, setVenues] = useState([]);
+  const [searchText, setSearchText] = useState('');
+  const [searchCity, setSearchCity] = useState('');
+  const [imageLinks, setImageLinks] = useState([]);
+  const baseUrl = 'http://localhost:3001';
+
+  useEffect(() => {
+    // eslint-disable-next-line array-callback-return
+    venues.map((venue, i) => {
```

Comment:
use `Array.prototype.forEach` if you don't want the array that `.map()` returns. Then you can remove the comment.